support-frontend/assets/pages/contributions-landing/contributionsLandingReducer.ts

```diff
@@ -50,13 +50,6 @@ export interface StripePaymentRequestButtonData {
 	paymentError: ErrorReason | null;
 }
 
-export interface Stripe3DSResult {
-	error?: Record<string, any>;
-	paymentIntent: {
-		id: string;
-	};
-}
```

Comment:
I was able to swap out all uses of this type with stripe's `PaymentIntentResult` type, which is just a better version of this one!